README.md

```diff
@@ -51,31 +51,37 @@ For more details on the e2e tests, see [test/e2e/README.md](test/e2e/README.md).
 
 **Main branch:**
 
-- **/test-centos-e2e-integration-main** runs CAPM3 e2e integration tests with
-  CAPM3 main branch and BMO **main** branch on Centos
-- **/test-ubuntu-integration-main** runs ansible integration tests with CAPM3
-  main branch and BMO **main** branch on Ubuntu
+- **/test metal3-centos-e2e-integration-test-main** runs CAPM3 e2e integration
+  tests with CAPM3 main branch and BMO **main** branch on Centos
+- **/test metal3-ubuntu-e2e-integration-test-main** runs ansible integration
+  tests with CAPM3 main branch and BMO **main** branch on Ubuntu
 
 **Release-0.5 branch:**
 
-- **/test-centos-e2e-integration-release-1-6** runs CAPM3 e2e integration tests
-  with CAPM3 release-1.6 branch and BMO **release-0.5** branch on Centos
-- **/test-ubuntu-integration-release-1-6** runs ansible integration tests with
-  CAPM3 release-1.6 branch and BMO **release-0.5** branch on Ubuntu
+- **/test metal3-centos-e2e-integration-test-release-1-6** runs CAPM3 e2e
+  integration tests with CAPM3 release-1.6 branch and BMO **release-0.5**
+  branch on Centos
+- **/test metal3-ubuntu-e2e-integration-test-release-1-6** runs ansible
+  integration tests with CAPM3 release-1.6 branch and BMO **release-0.5**
+  branch on Ubuntu
 
 **Release-0.4 branch:**
 
-- **/test-centos-e2e-integration-release-1-5** runs CAPM3 e2e integration tests
-  with CAPM3 release-1.5 branch and BMO **release-0.4** branch on Centos
-- **/test-ubuntu-integration-release-1-5** runs ansible integration tests with
-  CAPM3 release-1.5 branch and BMO **release-0.4** branch on Ubuntu
+- **/test metal3-centos-e2e-integration-test-release-1-5** runs CAPM3 e2e
+  integration tests with CAPM3 release-1.5 branch and BMO **release-0.4**
+  branch on Centos
+- **/test metal3-ubuntu-e2e-integration-test-release-1-5** runs ansible
+  integration tests with CAPM3 release-1.5 branch and BMO **release-0.4**
+  branch on Ubuntu
 
 **Release-0.3 branch:**
 
-- **/test-centos-e2e-integration-release-1-4** runs CAPM3 e2e integration tests
-  with CAPM3 release-1.4 branch and BMO **release-0.3** branch on Centos
-- **/test-ubuntu-integration-release-1-4** runs ansible integration tests with
-  CAPM3 release-1.4 branch and BMO **release-0.3** branch on Ubuntu
+- **/test metal3-centos-e2e-integration-test-release-1-4** runs CAPM3 e2e
```

Comment:
Replace this with 0.6 branch, 0.3 is no longer supported or CI tested.
/hold

Reply:
Do we have release-1.7 test running now ?

Reply:
done